modules/storage/src/storage-provider/index.ts

```diff
@@ -3,20 +3,26 @@ import { GoogleCloudStorage } from './providers/google';
 import { LocalStorage } from './providers/local';
 import { AzureStorage } from './providers/azure';
 import { AWSS3Storage } from './providers/aws';
+import { AliyunStorage } from './providers/aliyun';
 
 export function createStorageProvider(
-  provider: string,
+  provider: keyof StorageConfig,
   options: StorageConfig,
 ): IStorageProvider {
-  if (provider === 'google') {
-    return new GoogleCloudStorage(options);
-  } else if (provider === 'azure') {
-    return new AzureStorage(options);
-  } else if (provider === 'aws') {
-    return new AWSS3Storage(options);
-  } else {
-    return new LocalStorage(options);
+  switch (provider) {
```

Comment:
since this has been changed to a switch (kudos btw), use the default case instead of a final return on line 25.

Reply:
have fixed